### PR TITLE
Adds testcase to scale test code

### PR DESCRIPTION
**Pull request by metalcyanide**
Testcase for checking create, update and deletion of different type of ingress in a reboot during scale test.

### CODE REVIEW

File: tests/scaletest/scale_test.go [hunk 1]
```diff
@@ -73,7 +73,7 @@ var (
 	ingressType              string
 	numOfIng                 int
 	numOfLBSvc               int
-	numOfPodsForLBSvc        = 100
+	numOfPodsForLBSvc        = 98
```
Comment (monotosh-avi) [inline tests/scaletest/scale_test.go:76]: just a query, why is this changed to 98 from 100 ? :-)

**Review by monotosh-avi — APPROVED** [review]

File: tests/scaletest/scale_test.go (hunk 1 above)
Comment (metalcyanide) [reply tests/scaletest/scale_test.go:76]: I had to change it back, it's from something else I was checking out previously.

File: tests/scaletest/scale_test.go [hunk 2]
```diff
@@ -1080,99 +1098,70 @@ func TestMain(t *testing.M) {
...
+	testIngressTypeWithReboot(t, REBOOTCONTROLLER, MULTIHOST)
+}
+
+func TestAllIngressParallelCreationUpdationDeletionWithoutReboot(t *testing.T) {
+	ingTypes := []string{INSECURE, SECURE, MULTIHOST}
+	testMultipleIngressTypeWithReboot(t, REBOOTNONE, ingTypes)
+}
+func TestAllIngressParallelCreationUpdationDeletionWithAkoReboot(t *testing.T) {
+	ingTypes := []string{INSECURE, SECURE, MULTIHOST}
+	testMultipleIngressTypeWithReboot(t, REBOOTAKO, ingTypes)
```
Comment (RiyaGaniga-avi) [inline tests/scaletest/scale_test.go:1154]: As discussed, the `REBOOTAKO` should be passed as a pointer. And so does `REBOOTNODE` and `REBOOTCONTROLLER` in the following testcases.

**Review by RiyaGaniga-avi — APPROVED** [review]

**Comment by RiyaGaniga-avi** [comment]
build ako

**Comment by metalcyanide** [comment]
build ako

**Comment by metalcyanide** [comment]
build ako

**Review by monotosh-avi — APPROVED** [review]
